src/Internal/Payment/State/ProcessedState.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Class ProcessedState
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal\Payment\State;
+
+use WC_Payments_API_Payment_Intention;
+use WCPay\Core\Exceptions\Server\Request\Extend_Request_Exception;
+use WCPay\Core\Exceptions\Server\Request\Immutable_Parameter_Exception;
+use WCPay\Core\Exceptions\Server\Request\Invalid_Request_Parameter_Exception;
+use WCPay\Internal\Service\OrderService;
+use WCPay\Internal\Service\PaymentRequestService;
+
+/**
+ * This state is used when payment is completed on the server, and we need to update date on the plugin side.
+ */
+class ProcessedState extends AbstractPaymentState {
+
```

Comment:
Let's not add empty lines in the beginning of code blocks, they only make sense as separators between blocks of  code.